extensions/spark/kyuubi-spark-connector-hive/src/test/scala/org/apache/kyuubi/spark/connector/hive/HiveQuerySuite.scala

```diff
@@ -260,6 +269,90 @@ class HiveQuerySuite extends KyuubiHiveTest {
     }
   }
 
+  test("ORC filter pushdown") {
+    val table = "hive.default.orc_filter_pushdown"
+    withTable(table) {
+      spark.sql(s"""
```

Comment:
can you move the String definition to the next line, to avoid having a huge leading blank block?